.github/scripts/label_promoted_commits.py

```diff
@@ -54,7 +54,8 @@ def main():
     # Print commit information
     for commit in commits:
         print(f'Commit sha is: {commit.sha}')
-        match = pr_pattern.search(commit.commit.message)
+        pr_last_line = commit.commit.message.strip().split("\n")[-1]
+        match = re.search(rf"Closes .*#([0-9]+)", pr_last_line, re.IGNORECASE)
```

Comment:
why are you replicating the `pr_pattern` here instead of reusing it? how about
```py
pr_pattern = re.compile(r'Closes .*#([0-9]+)', re.IGNORECASE)
# ...
pr_last_line = commit.commit.message.strip().splitlines()[-1]
match = pr_pattern.search(pr_last_line)
```
?

Reply:
oh, I missed it. fixed as suggested with a minor change `pr_last_line = commit.commit.message.splitlines()[-1]`